js/coinex.js

```diff
@@ -614,6 +615,81 @@ module.exports = class coinex extends Exchange {
         return this.parseOrder (data, market);
     }
 
+    async cancelAllOrders (symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancellAllOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketId = market['id'];
+        let accountId = this.safeString2 (params, 'id', 'account_id');
```

Comment:
We don't unify the account id under the `id` parameter, so I would replace this line with `this.safeString (params, 'account_id')` (the `id` is not used for the same purpose in a unified way in any other exchange).